builders/build/eth-api/dev-env/brownie.md

```diff
@@ -52,6 +54,26 @@ You will need to install Brownie and create a Brownie project if you don't alrea
 
 ![Create Brownie project](/images/builders/build/eth-api/dev-env/brownie/brownie-1.png)
 
+!!!note
+    A common error while installing Brownie on Ubuntu is `pip seemed to fail to build package: pyyaml==5.4.1` This can be resolved by using pip3 as shown in Step 3 instead of pipx. 
+
+Error installing eth-brownie.
+
+Some possibly relevant errors from pip install:
+    error: subprocess-exited-with-error
+    AttributeError: cython_sources
+
+Error installing eth-brownie.
+
+pip seemed to fail to build package:
+    pyyaml==5.4.1
+
+Some possibly relevant errors from pip install:
+    error: subprocess-exited-with-error
+    AttributeError: cython_sources
+
+Error installing eth-brownie.
+
```

Comment:
This can all be removed. I like the format of error first, and then solution right after